R/bucket.R

```diff
@@ -1,11 +1,83 @@
-## hydromad: Hydrological Modelling and Analysis of Data
-##
-## Copyright (c) Felix Andrews <felix@nfrac.org>
-##
-
-## Bucket-type Soil Moisture Accounting models.
-## Bai et. al. (2009), Environmental Modelling and Software.
-## Model S2.
```

Comment:
Add back in the reference to Model S2 for the time being because it doesn't seem like it's mentioned anywhere else

Reply:
Done.